resources/views/pages/achievement/[achievement]/report-issue.blade.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+use function Laravel\Folio\{middleware, name};
+
+middleware(['auth', 'can:view,achievement']);
+name('achievement.report-issue');
+
+?>
+
+@props([
+    'achievement' => null, // Achievement
+])
+
+@php
+
+use App\Community\Enums\TicketType;
+
+$unlockedHardcore = false;
+$unlockedSoftcore = false;
+$unlockedManually = false;
+$hasHardcoreUnlocks = false;
+$hasSoftcoreUnlocks = false;
+$hasSession = false;
+
+$playerAchievements = request()->user()->playerAchievements()
+    ->join('Achievements', 'player_achievements.achievement_id', '=', 'Achievements.ID')
+    ->where('Achievements.GameID', $achievement->game->id)
+    ->get();
```

Comment:
It may be better to rely on the model relations over using an explicit join:

```php
// example
$user
  ->playerAchievements()
  ->whereHas("achievement.game", function ($query) use ($achievement) {
    $query->where("id", $achievement->game->id);
  })
  ->with("achievement.game")
  ->get();
```

Note in this example, no column names are actually used (even `GameID` is abstracted away). This means when a column gets renamed to implement one of the many TODOs in our models, this won't break.

The response shape of this query is going to be slightly different than the current implementation.

Reply:
added a `scopeForGame(Game)` to both `PlayerAchievement` and `PlayerGame`.